arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -438,38 +486,39 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
-            "name": "mediaServiceName",
-            "description": "Name of the Media Service.",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "pattern": "^[a-z0-9]",
-            "minLength": 3,
-            "maxLength": 24
+            "$ref": "#/parameters/mediaResourceNameParameter"
           },
           {
             "name": "SyncStorageKeysInput",
-            "description": "Properties needed to sycnronize the keys for a storage account to the Media Service.",
+            "description": "Properties needed to synchronize the keys for a storage account to the Media Service.",
             "in": "body",
             "required": true,
             "schema": {
               "$ref": "#/definitions/SyncStorageKeysInput"
             }
           }
-        ]
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The keys for the storage account were synchronized."
+          },
+          "default": {
+            "description": "Synch Media Service storage keys failed.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        }
       }
     }
   },
   "definitions": {
     "ApiEndpoint": {

```

Comment:
why have you not provided examples for model definitions?
As per the [specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#patterned-objects-9) (once the page loads, please scroll up a little).

example | 	Any	 |   A free-form property to include an example of an instance for this schema.